docs/config/server-options.md

```diff
@@ -83,7 +83,7 @@ Configure custom proxy rules for the dev server. Expects an object of `{ key: op
 
 Note that if you are using non-relative [`base`](/config/shared-options.md#base), you must prefix each key with that `base`.
 
-Uses [`http-proxy`](https://github.com/http-party/node-http-proxy). Full options [here](https://github.com/http-party/node-http-proxy#options).
+Extends [`http-proxy`](https://github.com/http-party/node-http-proxy) (full options [here](https://github.com/http-party/node-http-proxy#options)). Additional options are [here](https://github.com/vitejs/vite/blob/main/packages/vite/src/node/server/middlewares/proxy.ts#L13).
```

Comment:
Thanks! I might simplify by just linking directly to the options. The word "full" is a bit confusing, so that would also get rid of it which I think would make things clearer
```suggestion
Extends [`http-proxy`](https://github.com/http-party/node-http-proxy#options). Additional options are [here](https://github.com/vitejs/vite/blob/main/packages/vite/src/node/server/middlewares/proxy.ts#L13).
```

Reply:
SGTM